website/src/components/NavbarPopper.vue

```diff
@@ -0,0 +1,154 @@
+<template>
```

Comment:
Was this component being used? I couldn't see where it was being used yet. If not, kindly leave a comment so that someone else working on the file may know what it's for :D